internal/datatransfer/s3/sync.go

```diff
@@ -107,5 +107,9 @@ func createHttpClient(caBundle []byte) *http.Client {
 
 func (s *Sync) SyncPath(sourcePath, targetPath string) error {
 	target := fmt.Sprintf("s3://%s/%s", s.bucketName, targetPath)
-	return s.s3SyncManager.Sync(sourcePath, target)
+	err := s.s3SyncManager.Sync(sourcePath, target)
+	if err != nil {
+		return fmt.Errorf("error while syncing path to s3: %v", err)
```

Comment:
There is already `error while` [here](https://github.com/project-flotta/flotta-device-worker/blob/main/internal/datatransfer/monitor.go#L285). Please remove it from here or there.

Reply:
ok, I removed my message, now I only return the error itself